.github/workflows/galata.yml

```diff
@@ -108,8 +108,8 @@ jobs:
       - name: Checkout demo project
         uses: actions/checkout@v4
         with:
-          repository: jupyterlab/jupyterlab-demo
-          ref: master
+          repository: jtpio/jupyterlab-demo
+          ref: audio-video
```

Comment:
This currently depends on https://github.com/jupyterlab/jupyterlab-demo/pull/150.

Ideally, https://github.com/jupyterlab/jupyterlab-demo/pull/150 should first be merged so we can revert this change back to depending on `jupyterlab/jupyterlab`.